Program.cs

```diff
@@ -8,10 +12,20 @@ public static async Task Main()
             var config = new ServiceConfig("TestConfig.env");
 
             // Create the client
-            var client = RepositoryApiClient.CreateFromAccessKey(config.ServicePrincipalKey, config.AccessKey);
+            IRepositoryApiClient client;
+            client = RepositoryApiClient.CreateFromAccessKey(config.ServicePrincipalKey, config.AccessKey);
 
-            // Get a list of repository names
-            var repoNames = await GetRepoNames(client);
+            if (config.AuthorizationType.Equals("LfdsUsernamePassword", StringComparison.OrdinalIgnoreCase))
+            {
+                client = RepositoryApiClient.CreateFromLfdsUsernamePassword(config.Username, config.Password, config.Organization, config.RepositoryId, config.BaseUrl);
+            }
+            else if (config.AuthorizationType.Equals("AccessKey", StringComparison.OrdinalIgnoreCase))
+            {
+                client = RepositoryApiClient.CreateFromAccessKey(config.ServicePrincipalKey, config.AccessKey);
+            }
+
+            // Get a list of repository names (currently not available)
+            // var repoNames = await GetRepoNames(client);
```

Comment:
let's leave these lines for getting the repo names uncommented. This will work with the cloud repository client